src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -753,22 +753,49 @@ class NBDListIterator {
   }
 };
 
-static int load_module(Config *cfg)
+static int load_module(Config *cfg, bool remove_module)
 {
   ostringstream param;
   int ret;
 
-  if (cfg->nbds_max)
-    param << "nbds_max=" << cfg->nbds_max;
+  // will set it to 0 for netlink method to fix tracker#47560
```

Comment:
nit: I don't like very much when a tracker issue is referenced in a comment. 

If you think it is important to have a comment describing the reason why "nbds_max=0" should be set for netlink, then I would add the full explanation.

BTW, isn't "nbds_max=0" equivalent of just not specifying this  parameter at all?

Reply:
> BTW, isn't "nbds_max=0" equivalent of just not specifying this parameter at all?

I think it defaults to 16 if not specified.

Reply:
Yeah, if the kernel module doesn't see use have passed down the `nbds-max=X` parameter, it will just use the defualt `16`, or it will reset it if it is `>= 0`.

So for netlink method we should explicitly set it to 0, and for the ioctl method if user doesn't specify it we must ignore it.